### PR TITLE
fix(babel-preset-codecademy): remove loose option

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
### Overview

Removes `loose: true` from our class properties plugin so that everything is `loose: false`.

See http://2ality.com/2015/12/babel6-loose-mode.html for why I'm hoping to get us to `false`.

### PR Checklist

- ~[ ] Related to designs:~
- [x] Related to JIRA ticket: WEB-547
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/babel-preset-codecademy/index.js [hunk 1]
```diff
@@ -49,12 +49,7 @@ module.exports = (api, { type = PACKAGE_LIBRARY } = {}) => {
     plugins: [
       require('@babel/plugin-transform-destructuring').default,
       [require('@babel/plugin-proposal-decorators'), { legacy: true }],
-      [
-        require('@babel/plugin-proposal-class-properties').default,
-        {
-          loose: true,
```
Comment (JoshuaKGoldberg) [inline packages/babel-preset-codecademy/index.js:55]: @jakemhiller do you remember if there was context around setting this to `true` that I should know about?

I'm also wondering if we can get rid of this now that we haven't supported IE for a while...

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@4.0.2-alpha.7cc455.0<br/>@codecademy/tracking@0.13.2-alpha.7cc455.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6138f41137a2e100c1631847--gamut-preview.netlify.app>https://6138f41137a2e100c1631847--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6138f41137a2e100c1631847>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]

File: packages/babel-preset-codecademy/index.js (hunk 1 above)
Comment (jakemhiller) [reply packages/babel-preset-codecademy/index.js:55]: basically, just b/c it was what most people were doing at the time, and the idea that the resulting code was potentially faster and simpler seemed more important than compatibility 🤷 

are there a lot of code changes needed? i wouldn't assume so

**Review by jakemhiller — APPROVED** [review]

File: packages/babel-preset-codecademy/index.js (hunk 1 above)
Comment (JoshuaKGoldberg) [reply packages/babel-preset-codecademy/index.js:55]: I tried this out locally and couldn't find any issues. Us moving away so strongly from classes for components is probably helpful... I'll set up a preview environment.

Comment (JoshuaKGoldberg) [reply packages/babel-preset-codecademy/index.js:55]: Oh, right, I already did 😆  https://github.com/codecademy-engineering/Codecademy/pull/26949

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>babel-preset-codecademy@4.0.2-alpha.003966.0<br/>@codecademy/tracking@0.14.1-alpha.003966.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://613b7dd651f3c500926335b8--gamut-preview.netlify.app>https://613b7dd651f3c500926335b8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/613b7dd651f3c500926335b8>Deploy Logs</a></p>

**Pull request merged**